### DISCUSSION TITLE
Welcome to Area-Occupancy-Detection Discussions!

### DISCUSSION THREAD

**Discussion by Hankanman** [Hankanman/Area-Occupancy-Detection#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by G2740** [comment]
I just wanted to say thank you. Just downloaded a couple hours ago, star given.

I've been working on Bayesians for a while, HA recently made Bayesion a native GUI integration.

Your work is also or seems to be Bayesian with extra features.

My motion sensors are all Aqara P1.

Motion clears in 30 seconds (adjustable between 1 and 200 at battery cost)

Built in occupancy rimer clears after 10 minutes. (Probably not adjustable)🤷.

So in my first test Area Occupancy Detection, the default times I lowered significantly. 20 and 30 seconds. Does provide me with a variable time if I need more...or less.

I'll keep playing and if anything pops up or seems worth sharing, I'll holler back here.

Thanks 👍 Nice integration.

**Comment by G2740** [comment]
I really like Area Occupancy Detection. I've set up several rooms, just a motion sensor so far, and a template binary sensor that detects if an input boolean toogle is on, added automations that add 5 minutes extra toggle hold time. (All need tweaks)

As theyre triggered, Aqara motion sensors clear in 30 seconds, default. Motion clear might lower occupancy probability a little, I'm no Bayesian expert. Lol

As they clear, input booleans are already toggled, five minute holds are ongoing, then AOD decay kicks in. I could be wrong 😂.

With the new GUI Bayesian integration and YAML Bayesian I'd tried to create an Elderly in Distress Bayesian. Difficult to wrap my head around observations etc... 😵‍💫

So, with my AOD rooms, I also added house, chose social, added every motion sensor, (8) and every template holds binaries for a person wondering the house.

It seems to work well, and built in decay, as each room clears, 5 minute delays, probs all at 50.

House Bayesian, finally starts slowly counting down from 100, taking a guess, about 25ish+- minutes to below probability of 50 indicating possible distress, send notification. (See tweaking)

I love messing with Bayesians, tagged the dev (@HarvsG) who does the GUI Bayesion integration work and conversion from YAML for HAOS.

Hopefully Area Occupancy Detection gets around, I'll post and link as I am able. More users of Bayesian will help increase and make sure it doesn't get deprecated. Posted a pic, not sure its allowed here.🤷 Keep up the good work @Hankanman 

![Screenshot_20251102_155334_Home Assistant.jpg](https://github.com/user-attachments/assets/2181ce21-2c55-4d8c-9e7c-4c2d1e4fdda5)

**Reply by HarvsG** [reply]
Hi, thanks for the tag, how can I help?

**Reply by G2740** [reply]
Just bringing this integration to your attention, in case you were not aware of it. It might help keep Bayesian in HA long term.

> Hi, thanks for the tag, how can I help?

**Reply by HarvsG** [reply]
Does it actually use the Bayesian integration or is it a replacement?

**Reply by G2740** [reply]
The developer can answer that. I'm just playing with the integration.

To me it's more of a front end to Bayesian. Docs are fairly extensive, if you feel like reading.

@Hankanman ? Any comments?


> Does it actually use the Bayesian integration or is it a replacement?

**Reply by Hankanman** [reply]
It is intended as a replacement, essentially it uses the same bayesian principles but it calculates the probability for each sensor and prior for the user using historical data in HA so you just give it some input sensors and it gives you the probability without needing to do the maths it also adapts as the situation in your home changes and dynamically updates the prior based on time of day and day of week. Much more in the works like activity detection and an area considering adjacent areas

**Reply by Hankanman** [reply]
It did all start with the bayesian integration i was using it to and basically was too lazy to update probs and prior correctly wanted it dynamic
